src/sql/util.py

```diff
@@ -157,6 +157,108 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates=[]
+) -> bool:
+    """
+    Raises UsageError when duplicate arguments are passed to magics.
+    Returns true if no duplicates in arguments or aliases.
+
+    Parameters
+    ----------
+    magic_execute
+        The execute method of the magic class.
+    cmd_from
+        Which magic class invoked this function. One of 'sql', 'sqlplot' or 'sqlcmd'.
+    args
+        The arguments passed to the magic command.
+    allowed_duplicates
+        The duplicates that are allowed for the class which invoked this function.
```

Comment:
`The duplicates arguments that are allowed`

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R175-R177)